src/Modules/Legacy/Visualization/ShowAndEditDipoles.cc

```diff
@@ -212,7 +212,7 @@ void ShowAndEditDipoles::loadData()
   }
 
   bool zeroVector = false;
-  for(int i = 0; i < scale_.size(); i++)
+  for(uint i = 0; i < scale_.size(); i++)
```

Comment:
Can you use standard size_t here? uint is a compiler-specific typedef

Reply:
Good point

Reply:
I changed all ints to size_t since they were all positive and many were used to access arrays we were using size_t for.